### PR TITLE
Audio or Video Support for campaigns

### PR DESCRIPTION
**Pull request by SridharBa-Kore**

### CODE REVIEW

File: examples/esm/chat/index.html [hunk 1]
```diff
@@ -16,6 +16,8 @@
     </head>
 
     <body>
```
Comment (VenkateswararaoVe-Kore) [inline examples/esm/chat/index.html:18]: Remove these changes from PR

File: examples/esm/chat/js/index_chat.js [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 //OPTION #1
 let botOptions=chatConfig.botOptions;
 
-botOptions.JWTUrl = "PLEASE_ENTER_JWTURL_HERE"; 
+/* botOptions.JWTUrl = "PLEASE_ENTER_JWTURL_HERE"; 
 botOptions.userIdentity = 'PLEASE_ENTER_USER_EMAIL_ID';// Provide users email id here
 botOptions.botInfo = { name: "PLEASE_ENTER_BOT_NAME", "_id": "PLEASE_ENTER_BOT_ID" }; // bot name is case sensitive
 botOptions.clientId = "PLEASE_ENTER_CLIENT_ID";
-botOptions.clientSecret = "PLEASE_ENTER_CLIENT_SECRET";
+botOptions.clientSecret = "PLEASE_ENTER_CLIENT_SECRET"; */
```
Comment (VenkateswararaoVe-Kore) [inline examples/esm/chat/js/index_chat.js:13]: Remove these changes from PR in index_chat file

File: examples/esm/chat/js/index_chat.js [hunk 3]
```diff
@@ -52,7 +58,18 @@ https://developer.kore.ai/docs/bots/sdks/user-authorization-and-assertion/
 //   }
   
 //   chatWindowInstance.templateManager.installTemplate(new customTemplateComponent())
-chatWindowInstance.installPlugin(new AgentDesktopPlugin());
+// chatWindowInstance.installPlugin(new AgentDesktopPlugin());
+var agentDesktopPluginInstance=new AgentDesktopPlugin();
```
Comment (VenkateswararaoVe-Kore) [inline examples/esm/chat/js/index_chat.js:62]: Comment pwc plugin addition code

File: src/components/chatwindow/chatWindow.ts [hunk 4]
```diff
@@ -1900,7 +1900,7 @@ generateMessageDOM(msgData?:any){
       return "";
     }
     if (!messageHtml && msgData && msgData.message && msgData.message[0]) {
-      messageHtml = getHTML(Message, msgData, me); 
+      messageHtml = getHTML(Message, msgData, me); // use this for component html render
```
Comment (VenkateswararaoVe-Kore) [inline src/components/chatwindow/chatWindow.ts:1903]: Remove these changes from PR in chawindow file

File: src/components/chatwindow/config/kore-config.ts [hunk 5]
```diff
@@ -3,7 +3,7 @@ import BrandingJSON from '../sass/brandingJSON'; // To do
 var chatConfig:any={};
```
Comment (VenkateswararaoVe-Kore) [inline src/components/chatwindow/config/kore-config.ts:3]: remove these changes in config file

File: examples/esm/chat/index.html (hunk 1 above)
Comment (SridharBa-Kore) [reply examples/esm/chat/index.html:18]: @VenkateswararaoVe-Kore This PR Is not ready for review. will update you when it is ready. Please don't review until informed.

**Comment by SridharBa-Kore** [comment]
Please don't review this PR.

**Comment by SridharBa-Kore** [comment]
@RajasekharBa-Kore @VenkateswararaoVe-Kore @KambagiriswamyBo-Kore Please review this PR.

File: src/templatemanager/base/chatWidgetHeader/chatWidgetHeader.tsx [hunk 6]
```diff
@@ -142,15 +142,15 @@ export function ChatWidgetHeader(props: any) {
...
                         </svg>
                 </button> }
-                { brandingInfo.header.buttons.close.show && <button title="Close" className="btn-action btn-action-close" type="button" aria-label="Close Chat">
+                { brandingInfo.header?.buttons?.close?.show && <button title="Close" className="btn-action btn-action-close" type="button" aria-label="Close Chat">
```
Comment (VenkateswararaoVe-Kore) [inline src/templatemanager/base/chatWidgetHeader/chatWidgetHeader.tsx:153]: remove these null checks. already added condition. @SridharBa-Kore

File: src/templatemanager/base/chatWidgetHeader/chatWidgetHeader.tsx
Comment (VenkateswararaoVe-Kore) [inline src/templatemanager/base/chatWidgetHeader/chatWidgetHeader.tsx:1]: @SridharBa-Kore You can remove null checks here. already added condition.

File: src/templatemanager/base/chatWidget/chatWidget.scss
Comment (VenkateswararaoVe-Kore) [inline src/templatemanager/base/chatWidget/chatWidget.scss:1]: @SridharBa-Kore I think you can move this css to agent desktop plugin css file. Please check once.

**Comment by SridharBa-Kore** [comment]
@VenkateswararaoVe-Kore @RajasekharBa-Kore Can you please review.

**Pull request merged**